client/lib/geo-find-me.js

```diff
@@ -3,14 +3,14 @@ export default function geoFindMe() {
     const lat = position.coords.latitude;
     const long = position.coords.longitude;
     const KEY = 'AIzaSyDmADdAoHWHYXYsnAe1YAVaPgnlR6Fohow';
-    let address = '';
     fetch(
-      `https://maps.googleapis.com/maps/api/geocode/json?latlng=${lat},${long}&key=${KEY}`
+        `https://maps.googleapis.com/maps/api/geocode/json?latlng=${lat},${long}&key=${KEY}`
```

Comment:
Instead of hard-coding your api key, you should use an environment variable. Use the EnvironmentPlugin

https://webpack.js.org/concepts/plugins/#usage
https://webpack.js.org/plugins/environment-plugin/

Be sure to restart your dev script.